browser/js/checkout/checkout.js

```diff
@@ -35,10 +42,30 @@ app.controller('CheckoutCtrl', function($scope, cartItems, AuthService, Card, Ca
                 return Order.createOrderDetails(order.orderDetails);
             })
             .then(() => {
-                var user = AuthService.getLoggedInUser();
+
                 $scope.cartItems = {};
-                if(user) { Cart.clearCartUser() 
+                if (user) {
+                    Cart.clearCartUser()
                 } else { Cart.clearCartVisitor() }
             })
+        }
+    }
+
+    $scope.createOrder = function(order) {
+        order.orderSummary.cardId = order.card.id;
+
+        var $form = {
+            'number': order.card.number,
+            'exp_month': order.card.exp_month,
+            'exp_year': order.card.exp_year,
+            'cvc': order.card.cvc
+        }
+
+        AuthService.getLoggedInUser()
+            .then(userLoggedIn => {
+                $scope.user = userLoggedIn;
+                return Stripe.card.createToken($form, responseHandler)
+            })
+            .catch(console.log);
```

Comment:
I don't think you can do this on the frontend.  It'd have to be like `console.log.bind(console)`